vercel.json

```
@@ -1438,6 +1438,62 @@
     {
       "source": "/docs/guides/migrate/deployment/deploy-database-changes-with-prisma-migrate",
       "destination": "/docs/guides/deployment/deploy-database-changes-with-prisma-migrate"
+    },
+    {
+      "source": "/docs/guides/deployment/deployment",
+      "destination": "/docs/guides/deployment/deploying-prisma"
+    },
+    {
+      "source": "/docs/guides/deployment/deployment-guides/deploying-to-heroku",
+      "destination": "/docs/guides/deployment/traditional-deployments/deploying-to-heroku"
```

(Commenting here instead of individual file as this is a nice overview of the planned URLs)

These URLs could also just be
```suggestion
      "destination": "/docs/guides/deployment/traditional/deploying-to-heroku"
```
(Same for all other URLs of course)